skipped_tests.toml

```diff
@@ -153,6 +154,11 @@ functional = [
 	"tests/pytests/functional/runners/test_winrepo.py::test_legacy_update_git_repos",
 	"tests/pytests/functional/runners/test_winrepo.py::test_update_git_repos",
 	"tests/pytests/functional/states/test_pip_state.py::test_pip_installed_name_test_mode",
+
+	"tests/pytests/functional/modules/test_x509_v2.py::test_create_private_key[ed25519]",  # Needs investigation: Fails only with Salt Bundle - cryptography.exceptions.UnsupportedAlgorithm: ed25519 is not supported by t...
+	"tests/pytests/functional/modules/test_x509_v2.py::test_create_private_key[ed448]",  # Needs investigation: Fails only with Salt Bundle - cryptography.exceptions.UnsupportedAlgorithm: ed448 is not supported by thi...
+	"tests/pytests/functional/modules/test_x509_v2.py::test_create_private_key_with_passphrase[ed25519]",  # Needs investigation: Fails only with Salt Bundle - cryptography.exceptions.UnsupportedAlgorithm: ed25519 is not supported by t...
+	"tests/pytests/functional/modules/test_x509_v2.py::test_create_private_key_with_passphrase[ed448]",  # Needs investigation: Fails only with Salt Bundle -  cryptography.exceptions.UnsupportedAlgorithm: ed448 is not supported by thi...
```

Comment:
The actual root cause of these tests fails is the missing algorithms in openssl library on the client, so there is no way to fix it on the `salt` level. I think these failing tests were fixed with https://github.com/openSUSE/salt/pull/646, are they still failing?

Reply:
I see these tests still failing for Leap 15.5, where the mentioned fixes should be available: https://ci.suse.de/user/manager/my-views/view/Salt%20Shaker/job/manager-salt-shaker-products-testing-leap155-bundle/

Host: salt-shaker-products-testing-leap155-bundle.mgr.suse.de

I haven't debug the tests yet.

Reply:
I see what's the problem. I haven't fixed all the tests this way, these two are marked as slow and seems I missed them. But better to fix them instead of adding as skip, I think.

Reply:
Fixed in https://github.com/openSUSE/salt/commit/7daf461528c90776b8f865cd58d20e23bd5b6f3f - we can remove this from the PR.